### PR TITLE
small lil pair of bugfixes

### LINKED ISSUE #8611: Advanced body scanner reporting NaN/spurious data
#### Brief description of the issue
Scanning a patient in the advanced scanner (patient was teshari) returned NaN weight. Temperature possibly also spurious
#### What you expected to happen
Absence of a NaN error
#### What actually happened
Screenshot 2022-05-06 032427

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

### LINKED ISSUE #8642: Emote activated implants don't function
#### Brief description of the issue
When attempting to use implants that are activated using *[emote] (such as a freedom implant) do not activate, instead you just emote with no additional effects.

#### What you expected to happen
Using the emote listed in my notes and being immediately released from cable restraints/handcuffs (using the freedom implant example.)

#### What actually happened
Just the emote.

#### Steps to reproduce
Either print freedom implant from protolathe or buy any emote-activated implant from traitor uplink.
Implant yourself.
Use cable restraints/handcuffs on yourself.
Use activation emote.

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add**: Tested on freedom implant and compressed matter implant, implanted in multiple body parts, using both cable restraints and handcuffs when testing freedom implants.

### PR DESCRIPTION
**Pull request by Ilysen**
hi! first-time contributor here; I hope I'm doing this right. it seems like there's not a PR template, so I'll be kitbashing one myself. I'm uncertain if fix PRs should be atomized to a degree, or if they're fine as bundles, so I can do either going forward.

#### purpose

Fixes #8611
Fixes #8642 

#### details

***Body scanner data:*** this seems to be due to some lines that were left in with the tgui port from vorestation that I can, god knows, only assume relate to things that are unique to their codebase. I removed the incorrect lines from the .js file and recompiled tgui.

***Emote implants:*** instead of checking all implants in the source mob's contents (which didn't account for nested contents, and so was missed because implants were now housed in organs), the logic now checks the implants in every external organ in the source mob. this breaks for the base-level `/mob` proc, so I moved it to a human-specific override so as not to clutter the base-level proc with type-specific logic.

#### testing

for the first bug: I hopped into a body scanner after recompiling the fixes, and verified that the interface was showing properly now.

for the second bug: I gave myself an uplink and tried out a few different implants, making sure that they indeed didn't work before applying the fix. after applying the fix, I tried again, and verified that the implants were now working again.

#### screenshots

<details><summary>the body scanner works correctly! note the lack of incorrect info in the summary</summary>

HdIcPzRvjI
</details>

<details><summary>you can trigger implants again! (ft. lighting system cameo)</summary>

dreamseeker_Z4ht8PVA8A
</details>

### CODE REVIEW

**Comment by Atermonera** [comment]
Correct, there is no template.
Generally, bundling things together is fine, it only really becomes an issue if something is contentious, at which point we may ask that particular changes be split into separate PRs to unblock merging of the less-controversial stuff while discussion happens around the parts people are arguing about. The way you've split up commits to address different issues is a good approach.

**Review by Atermonera — APPROVED** [review]

**Comment by Atermonera** [comment]
In terms of establishing expectations for reviews, it's mostly myself and Spookerton, and we don't always have time to review things in a timely manner. Technically there's a rule that we're not supposed to merge things for at least 24h after they go up, buuuuuut we can be easily persuaded to ignore it (Such as, when I chance to look at the github).

I personally tend to merge things oldest->newest, so if you put something up that should replace something older, please bug me about it because I am historically bad at noticing those things before I slap merge.

**Comment by Ilysen** [comment]
good to know. thanks for the heads up ^^

**Pull request merged**
